packages/mds-db/processors.ts

```diff
@@ -12,6 +12,8 @@ import schema from './schema'
 import { vals_sql, cols_sql, vals_list, logSql, SqlVals } from './sql-utils'
 import { getWriteableClient, makeReadOnlyQuery } from './client'
 
+const tenantId = process.env.TENANT_ID ? process.env.TENANT_ID : 'mds'
+
```

Comment:
Do we actually want `mds.telemetry` or `tenant.event` written as the value of the `type` column in the database tables since the database is already segregated by tenant? Feels like our code should probably just prepend and remove the tenant id for the purpose of routing such that the rest of the code just operates on `event` and `telemetry` types. That way, all of the tenant ID manipulation could be confined to just a couple of places ... `writeCloundEvent` (add the prefix) and `EventServer` remove the prefix before passing the type/data to the handlers. Otherwise, I fear it will me too easy to forget to account for the tenantId in future scenarios.

Reply:
yeah @evanxadkins and i discussed this very point last friday. i would advocate we do not put routing-information in the db. doing so would likely require a data deployment data mutation.

Reply:
I proposed stripping the jurisdiction IDs off the events as they enter the system, it shouldn't be too difficult to make the changes, they're not in many places.

Reply:
Pretty sure it could be done in the event server. There's a `parseCloudEvent` function that everything goes through.

Reply:
That or perhaps in the knative trigger filter.

Reply:
That would be fine too although it appears the prefix is being added in code so doing it in knative feels a little asymmetrical. If there was a way to both add and remove the prefix in the trigger/filter and not have to do any manipulation in code that would be great.

Reply:
looking into this now: filter/remove tenantId from event/type upon trigger invocation

Reply:
ok ... was able to strip off the tenant-routing-identifier in the event-server prior to forwarding the message to the handler.

additionally, in my env i am not seeing errors in the mds-event-server:EventServer.parseCloudEvent() catch method at this time